### PR TITLE
doc: add FAQ to docs

### PR DESCRIPTION
**Pull request by rickycao-qy**
add FAQ to docs.

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -20,4 +20,7 @@
 - Specification
   - [Plugin Specification](/spec/plugin.md)
   - [Dataset Specification](/spec/dataset.md)
+- FAQ
```
Comment (yorkie) [inline docs/_sidebar.md:23]: We need to add guidance on this page for its sub pages.

File: docs/zh-cn/faq/pipcook-framework.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Frequently Asked Questions
```
Comment (WenheLI) [inline docs/zh-cn/faq/pipcook-framework.md:1]: This should be Chinese?

File: docs/zh-cn/faq/plugins.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# 常见问题
+
+### <div id="q1">在 yolov5 的链路中我怎样指定用哪张显卡进行训练?</div>
+你可以设置 `$CUDA_VISIBLE_DEVICES`  这个环境变量，它的值就是你的显卡编号，默认我们会使用 GPU:0 进行训练
+
+### <div id="q1">在 yolov5 的训练中我怎样指定仅使用 cpu 训练？</div>
+如果你的环境支持 GPU 训练，我们默认会使用 GPU。如果你想禁掉此功能，可以设置环境变量 `export CUDA_VISIBLE_DEVICES=""`
```
Comment (FeelyChau) [inline docs/zh-cn/faq/plugins.md:9]: All the environment variables should be set to daemon processor, I think we should explain that.

File: docs/_sidebar.md (hunk 1 above)
Comment (rickycao-qy) [reply docs/_sidebar.md]: done

File: docs/zh-cn/faq/pipcook-framework.md (hunk 2 above)
Comment (rickycao-qy) [reply docs/zh-cn/faq/pipcook-framework.md]: done

**Comment by rickycao-qy** [comment]
@WenheLI @yorkie Please have a check again

File: docs/zh-cn/faq/pipcook-framework.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# 常见问题
+
+## <div id="q1">我应该在哪里放置我的 pipeline 的配置文件?</div>
+Pipcook 会在后台启动服务，用户可以通过 cli 工具或者可视化工具访问服务。因此，我们对您的工作目录没有特别的要求，你可以在任意地方放置你的配置文件。只需要使用 `pipcook run <url>` 并且指定正确的配置文件路径就可以进行训练了
```
Comment (FeelyChau) [inline docs/zh-cn/faq/pipcook-framework.md:5]:
```suggestion
Pipcook 会在后台启动服务，用户可以通过 cli 工具或者可视化工具访问服务。因此，我们对您的工作目录没有特别的要求，你可以在任意地方放置你的配置文件。只需要使用 `pipcook run <url>` 并且指定正确的配置文件路径就可以进行训练了， url 参数支持本地路径或者远程 url。
```

File: docs/zh-cn/faq/index.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# 常见问题
```
Comment (yorkie) [inline docs/zh-cn/faq/index.md:1]: This should be consistent with the TOC :)

**Review by FeelyChau — CHANGES_REQUESTED** [review]

File: docs/zh-cn/faq/pipcook-framework.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# 常见问题
+
+## 我应该在哪里放置我的 pipeline 的配置文件?
+<a id="q1"></a>
```
Comment (yorkie) [inline docs/zh-cn/faq/pipcook-framework.md:4]: This should be removed.

File: docs/zh-cn/faq/pipcook-framework.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+Pipcook 会在后台启动服务，用户可以通过 cli 工具或者可视化工具访问服务。因此，我们对您的工作目录没有特别的要求，你可以在任意地方放置你的配置文件。只需要使用 `pipcook run <url>` 并且指定正确的配置文件路径就可以进行训练了
+
+## 为什么 Pipcook 安装非常缓慢？
+<a id="q2"></a>
```
Comment (yorkie) [inline docs/zh-cn/faq/pipcook-framework.md:8]: Ditto.

File: docs/zh-cn/faq/plugins.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# 常见问题
+
+### 在 yolov5 的链路中我怎样指定用哪张显卡进行训练?
+<a id="q1"></a>
```
Comment (yorkie) [inline docs/zh-cn/faq/plugins.md:4]: Ditto

File: docs/zh-cn/faq/pipcook-framework.md (hunk 6 above)
Comment (rickycao-qy) [reply docs/zh-cn/faq/pipcook-framework.md:4]: I add this as Anchor point. I think we should not remove it

Comment (yorkie) [reply docs/zh-cn/faq/pipcook-framework.md:4]: Markdown owns its anchor point for headings, why do we still use this instead?

Comment (rickycao-qy) [reply docs/zh-cn/faq/pipcook-framework.md:4]: To use heading anchor points, we need to write the whole heading contents after #, which is quite ugly, especially for Chinese version.

**Review by yorkie — APPROVED** [review]

**Review by WenheLI — DISMISSED** [review]
LGTM

**Review by FeelyChau — APPROVED** [review]
LGTM.

**Comment by rickycao-qy** [comment]
@yorkie  Following Feely's comments, add the explanation to the environment variable. Please check again~~

**Review by FeelyChau — APPROVED** [review]
LGTM.

**Review by yorkie — APPROVED** [review]

**Review by WenheLI — APPROVED** [review]
LGTM

**Pull request merged**